pkgs/tools/system/btop/default.nix

```diff
@@ -37,6 +39,11 @@ stdenv.mkDerivation rec {
     ${removeReferencesTo}/bin/remove-references-to -t ${stdenv.cc.cc} $(readlink -f $out/bin/btop)
   '';
 
+  postPhases = lib.optionals rocmSupport [ "postPatchelf" ];
+  postPatchelf = lib.optionals rocmSupport ''
+    patchelf --add-rpath "${lib.getLib rocmPackages.rocm-smi}/lib" $out/bin/btop
+  '';
```

Comment:
Wait, this is `optionalString :: Bool -> String -> String` not `optionals :: Bool -> List -> List`

Reply:
Right, second one should be `optionalString`